### PR TITLE
doc: the update for QAT Acceleration

### PR DESCRIPTION
**Pull request by qwren**
This patch update the documentation about QAT acceleration for encryption
and compression.

QATZip provide the interfaces for several compression algorithm, including
deflate, snapp and lz4, but currently only deflate can be accelerated by QAT
hardware. The support for snappy and lz4 in QAT hardware should be still in
the roadmap.

Signed-off-by: Qiaowei Ren <qiaowei.ren@intel.com>

### CODE REVIEW

File: doc/radosgw/qat-accel.rst [hunk 1]
```diff
@@ -38,8 +38,10 @@ The driver package can be downloaded from `Intel Quickassist Technology`_.
 2. The implementation for QAT based encryption is directly base on QAT API which
    is included the driver package. But QAT support for compression depends on
    QATzip project, which is a user space library which builds on top of the QAT
-   API. QATZip can support several compression algorithm, including deflate,
-   snappy, lz4, etc..
+   API. QATZip provide the interfaces for several compression algorithm, including
+   deflate, snapp and lz4, but currently only deflate can be accelerated by QAT
+   hardware. The support for snappy and lz4 in QAT hardware should be still in
+   the roadmap.
```
Comment (tchaikov) [inline doc/radosgw/qat-accel.rst:44]:
```suggestion
   API. Currently, QATZip speeds up gzip compression and decompression at the time of writing.
```

no need to mention the features still being planned, and i cannot find any published roadmap mentioning the support of snappy and/or lz4.

Comment (tchaikov) [reply doc/radosgw/qat-accel.rst:44]: also,

> The common tool class will be shared among zip, snappy, lz4 compressor
plugins, and can transparently accelerate the existing compression types.

IMHO, this is misleading. as it implies that snappy and lz4 compressors are accelerated by QATzip transparently. but they are not supported. could you update the "Implementation" section as well?

**Comment by tchaikov** [comment]
> doc: the update for QAT Acceleration

this title is not specific enough. could you change it to something like

> doc: update QATZip doc for removing snappy and lz4 support

File: doc/radosgw/qat-accel.rst (hunk 1 above)
Comment (qwren) [reply doc/radosgw/qat-accel.rst]: sure, I just fixed all above comments.

Comment (tchaikov) [reply doc/radosgw/qat-accel.rst]: @qwren could you address https://github.com/ceph/ceph/pull/36762#discussion_r475325290 ?

Comment (qwren) [reply doc/radosgw/qat-accel.rst]: > @qwren could you address [#36762 (comment)](https://github.com/ceph/ceph/pull/36762#discussion_r475325290) ?

@tchaikov looks like that I have removed the description about the features still being planned. you mean I still make more changes, right?

Comment (tchaikov) [reply doc/radosgw/qat-accel.rst]: @qwren yes. i'd suggest avoid mentioning snappy and lz4 at all. features not implemented is not interesting, IMHO. please see the suggested change i posted.

Comment (qwren) [reply doc/radosgw/qat-accel.rst]: Ok. I will update it.

**Review by tchaikov — APPROVED** [review]

**Pull request merged**